api/src/main/java/org/openmrs/module/attachments/AttachmentsConstants.java

```diff
@@ -32,8 +32,12 @@
 	
 	public static final String UPLOAD_ATTACHMENT_URL = "/rest/v1/" + ATTACHMENT_URI;
 	
+	public static final String OLD_UPLOAD_ATTACHMENT_URL = MODULE_BASE_URL + "/upload";
+	
 	public static final String DOWNLOAD_ATTACHMENT_URL = MODULE_BASE_URL + "/download";
 	
+	public static final String SEARCH_ATTACHEMENT_URL = "/rest/v1/" + ATTACHMENT_URI;
```

Comment:
`SEARCH_ATTACHEMENT_URL` → `ATTACHMENT_RESOURCE_URL` (mind the typo)

Reply:
can't understand what you mean in here.

Reply:
There was a typo in 'ATTACHEMENT' (it's 'ATTACHMENT') but anyway please name this constant `ATTACHMENT_RESOURCE_URL`. It is not only a search URL, it is also the URL for the resource itself in general.